yt/chyt/controller/internal/jupyt/controller.go

```diff
@@ -1,23 +1,32 @@
 package jupyt
 
 import (
+	"bytes"
 	"context"
-
+	"encoding/json"
+	"fmt"
 	"go.ytsaurus.tech/library/go/core/log"
 	"go.ytsaurus.tech/yt/chyt/controller/internal/strawberry"
 	"go.ytsaurus.tech/yt/go/ypath"
 	"go.ytsaurus.tech/yt/go/yson"
 	"go.ytsaurus.tech/yt/go/yt"
 	"go.ytsaurus.tech/yt/go/yterrors"
+	"io"
```

Comment:
Did you run `go fmt`?
My version sorts the import paths differently (system imports are separated from the `go.ytsaurus.tech` ones)

Reply:
It seems you have `goimports`, which sorts imports the way you expect. I installed it too.